docs/source/terms_and_model.rst

```diff
@@ -1147,3 +684,965 @@ The GA4GH computed identifier for these sets is
+   }
+
+.. _SequenceExpression:
+
+Sequence Expression
+@@@@@@@@@@@@@@@@@@@
+
+VRS provides several syntaxes for expressing a sequence,
+collectively referred to as *Sequence Expressions*. They are:
+
+* :ref:`LiteralSequence`: An explicit :ref:`Sequence`.
+* :ref:`DerivedSequence`: A sequence that is derived from a
+  :ref:`Sequencelocation`.
+* :ref:`RepeatedSequence`: A description of a repeating :ref:`Sequence`.
+
+
+.. _LiteralSequence:
+
+LiteralSequence
+###############
+
+A LiteralSequence "wraps" a string representation of a sequence for
+parallelism with other SequenceExpressions.
+
+**Computational Definition**
+
+An explicit expression of a Sequence.
+
+**Information Model**
+
+.. list-table::
+   :class: reece-wrap
+   :header-rows: 1
+   :align: left
+   :widths: auto
+
+   * - Field
+     - Type
+     - Limits
+     - Description
+   * - type
+     - string
+     - 1..1
+     - MUST be "LiteralSequence"
+   * - sequence
+     - :ref:`Sequence`
+     - 1..1
+     - The sequence to express
+
+
+.. _DerivedSequence:
+
+DerivedSequence
+###############
+
+Certain mechanisms of variation result from relocating and
+transforming sequence from another location in the genome.
+A *derived sequence* is a mechanism for expressing (typically
+large) reference subsequences specified by a :ref:`SequenceLocation`.
+
+**Computational Definition**
+
+An expression of a sequence that is derived from a referenced
+sequence location.
+
+**Information Model**
+
+.. list-table::
+   :class: reece-wrap
+   :header-rows: 1
+   :align: left
+   :widths: auto
+
+   * - Field
+     - Type
+     - Limits
+     - Description
+   * - type
+     - string
+     - 1..1
+     - MUST be "DerivedSequence"
+   * - location
+     - :ref:`SequenceLocation`
+     - 1..1
+     - The location describing the sequence
+   * - reverse_complement
+     - bool
+     - 1..1
+     - When True, indicates the derived sequence is a
+       reverse complement of the location sequence
+
+
+
+.. _RepeatedSequence:
+
+RepeatedSequence
+################
+
+*Repeated Sequence* is a class of sequence expression where a specified
+subsequence is repeated multiple times in tandem. Microsatellites are an
+example of a common class of repeated sequence, but repeated sequence can
+also be used to describe larger subsequence repeats, up to and including
+large-scale tandem duplications.
+
+**Computational Definition**
+
+An expression of a sequence comprised of a tandem repeating subsequence.
+
+**Information Model**
+
+.. list-table::
+   :class: reece-wrap
+   :header-rows: 1
+   :align: left
+   :widths: auto
+
+   * - Field
+     - Type
+     - Limits
+     - Description
+   * - type
+     - string
+     - 1..1
+     - MUST be "RepeatedSequence"
+   * - sequence
+     - :ref:`SequenceExpression`
+     - 1..1
+     - an expression of the repeating subsequence
+   * - count
+     - :ref:`CopyCount`
+     - 1..1
+     - the inclusive range count of repeated units
+
+
+.. _Feature:
+
+Feature
```

Comment:
Consider adding a NOTE to the Feature topic that describes the intention to have the Sequence Annotation group within GA4GH own the eventual models that represent the computational concepts that may be interchangeable for the Features below.